Billmike/More-Recipes
# Setup Backend Dependencies

# **What does this PR do?**
Setup Backend Dependencies for More-Recipes project
# **Description of Task to be completed?**
- Setup Nodejs
- Setup Sequelize
- Setup Expressjs
- Setup PostgreSQL
# **How should this be tested?**
- After cloning remote repo, cd into ```server``` folder
- run ```npm run start:dev```
- Navigate to ```localhost:8000``` in your favorite browser to see the result below.

![pr1](https://user-images.githubusercontent.com/24798364/29874311-3f655664-8d8e-11e7-9864-1b94fff9aa21.PNG)



# **What are the relevant pivotal tracker stories?**
#150670064
